editor/editor_dock_manager.h

```diff
@@ -77,6 +77,12 @@ class EditorDockManager : public Object {
 private:
 	friend class DockContextPopup;
 
+	enum TabStyle {
```

Comment:
```suggestion
	enum class TabStyle {
```
IIRC this is preferred nowadays.